lookout/style/format/features.py

```diff
@@ -229,84 +230,84 @@ def _classify_vnodes(self, nodes: Iterable[VirtualNode]) -> List[VirtualNode]:
                         cls = CLASS_INDEX[CLS_TAB]
                     else:
                         cls = CLASS_INDEX[CLS_SPACE]
-                    offset, line, col = node.start
-                    result.append(VirtualNode(
```

Comment:
line is used as a string afterwards so the latter usage conflicts with this one. Since you already use lineno later in the function to denote line number I think it's way better to rename actually.

Reply:
ok